modules/geoipbackend/geoipbackend.cc

```diff
@@ -121,7 +121,7 @@ static bool validateMappingLookupFormats(const vector<string>& formats)
   return true;
 }
 
-bool GeoIPBackend::loadDomain(const YAML::Node& domain, unsigned int id, GeoIPDomain& dom)
+bool GeoIPBackend::loadDomain(const YAML::Node& domain, unsigned int /* id */, GeoIPDomain& dom)
```

Comment:
If we don't use `id` it should probably go?

Reply:
Correct